templates/junit.tmpl

```diff
@@ -7,7 +7,15 @@
         </properties>
         {{- range .Matches }}
         <testcase classname="{{ .Artifact.Name }}-{{ .Artifact.Version }} ({{ .Artifact.Type }})" name="[{{ .Vulnerability.Severity }}] {{ .Vulnerability.ID }}">
-            <failure message="{{ .Artifact.Name }}: {{ .Vulnerability.ID }}" type="description">{{html .Vulnerability.Description }} {{ .Artifact.CPEs }} {{ .Vulnerability.DataSource }}</failure>
+            <failure message="{{ .Artifact.Name }}: {{ .Vulnerability.ID }}" type="description">
+<![CDATA[
+{{ html .Vulnerability.Description }}
+
+{{ .Artifact.CPEs }}
```

Comment:
Does it make sense to loop over the CPEs and put a newline? Right now it looks like this:

``` xml
        </testcase>
        <testcase classname="libcrypto3-3.5.0-r0 (apk)" name="[Medium] CVE-2025-4575">
            <failure message="libcrypto3: CVE-2025-4575" type="description">
<![CDATA[


[cpe:2.3:a:libcrypto3:libcrypto3:3.5.0-r0:*:*:*:*:*:*:* cpe:2.3:a:libcrypto3:libcrypto:3.5.0-r0:*:*:*:*:*:*:* cpe:2.3:a:libcrypto:libcrypto3:3.5.0-r0:*:*:*:*:*:*:* cpe:2.3:a:libcrypto:libcrypto:3.5.0-r0:*:*:*:*:*:*:*]

https://security.alpinelinux.org/vuln/CVE-2025-4575

]]></failure>
        </testcase>
```

which makes the CPEs a little hard to read.

Reply:
oh right, good catch, fixed in the next commit, if it looks good then please resolve the conversation.